delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/EventsRoutesSpec.scala

```diff
@@ -32,17 +32,14 @@ class EventsRoutesSpec extends BaseRouteSpec {
   private val identities = IdentitiesDummy(caller)
   private val asAlice    = addCredentials(OAuth2BearerToken("alice"))
 
-  private val acl       = Label.unsafe("acl")
   private val project   = Label.unsafe("project")
-  private val resources = Label.unsafe("resources")
+  private val resources = Label.unsafe("resource")
```

Comment:
The `resource` selector does not exist, it is just `resources` that must return all resource event (including schema, resolver, etc...)
Might be worth to check if it still works and if it is tested

Reply:
It's just that here the implementation of streamBy is filtering based on the type defined when defining the SSE, and it was `Project` and `Resource`. In fact there were several tests that were passing before but for the wrong reasons. And `project` was also in singular. Anyway, I will just make it plural to avoid any confusion.

Also I have changed the ACL test to be a test on resources sse so that will be covered. I will check in the integration tests